egs/wsj/s5/steps/nnet3/xconfig_to_configs.py

```diff
@@ -156,11 +156,26 @@ def write_config_files(config_dir, all_layers):
 
     config_basename_to_header = get_config_headers()
 
+    input_pairs = []
+
     for layer in all_layers:
         try:
             pairs = layer.get_full_config()
+            # we judge whether the layer is preconditioning lda by retained name 'lda'.
```

Comment:
Here, we start judge use LDA or not by retained name 'lda'. This is the only making sense method that i come up with.